torch/_dynamo/utils.py

```diff
@@ -299,6 +299,159 @@ def get_runtime_metrics_context() -> RuntimeMetricsContext:
     return _RUNTIME_METRICS_CONTEXT
 
 
+class CompileEventLogLevel(enum.Enum):
+    """
+    Enum that loosely corresponds with a "log level" of a given event.
+
+    CHROMIUM_EVENT: Logs only to tlparse.
+    COMPILE_EVENT: Logs to tlparse + PT2 Compile Events
+    COMPILATION_METRIC: Logs to tlparse, PT2 Compile Events, and dynamo_compile
```

Comment:
I wonder if it would make more sense to call this DYNAMO_COMPILE. Although internal, that name has already leaked in various places (like the name of the column passed to `dynamo_timed`). And you've even referenced dynamo_compile here.